packages/cli/src/commands/pull.ts

```diff
@@ -42,18 +42,29 @@ const help = () => {
 
   ${chalk.dim('Examples:')}
 
-  ${chalk.gray('–')} Pull the latest Project Settings from the cloud
+  ${chalk.gray(
+    '–'
+  )} Pull the latest Environment Variables and Project Settings from the cloud
+    and stores them in \`.vercel/.env.\${target}.local\` and \`.vercel/project.json\` respectively.
 
     ${chalk.cyan(`$ ${getPkgName()} pull`)}
     ${chalk.cyan(`$ ${getPkgName()} pull ./path-to-project`)}
-    ${chalk.cyan(`$ ${getPkgName()} pull --env .env.local`)}
-    ${chalk.cyan(`$ ${getPkgName()} pull ./path-to-project --env .env.local`)}
 
-  ${chalk.gray('–')} Pull specific environment's Project Settings from the cloud
+  ${chalk.gray('–')} Pull for a specific environment
 
     ${chalk.cyan(
       `$ ${getPkgName()} pull --environment=${getEnvTargetPlaceholder()}`
     )}
+
+  ${chalk.gray(
+    '–'
+  )} Pull the latest Environment Variables to a specific file. (Deprecated)
+    ${chalk.gray(
+      'If you want to download environment variables to a specific file, use `vercel env pull` instead.'
+    )}
+
+    ${chalk.cyan(`$ ${getPkgName()} pull --env .env.local`)}
```

Comment:
I tried `vc pull --env .env.local` on Vercel CLI 25 and it doesn't create the `.env.local`

Reply:
You're right--It doesn't work. I saw the reference `--env` in the command logic and assumed it was still actually used. I'll check that more explicitly next time.

I removed this bit of documentation and the reference to `--env`.